### ISSUE TITLE
Bioconda installation is confusing to call (ThermoRawFileParser.sh)

### ISSUE THREAD

**Issue by RalfG** [compomics/ThermoRawFileParser#84]
If installed with bioconda, the package's binary is called `ThermoRawFileParser.sh`, which means the program can only be started by calling `ThermoRawFileParser.sh`, which seems like a pretty confusing name to me. I would definitely remove the `.sh` extension, and maybe add a second binary called `thermorawfileparser`, so lower case also works.
```
❯ thermorawfileparser
zsh: command not found: thermorawfileparser
❯ ThermoRawFileParser
zsh: command not found: ThermoRawFileParser
❯ thermorawfileparser.sh
zsh: command not found: thermorawfileparser.sh
❯ ThermoRawFileParser.sh
-i, --input or -d, --input_directory: specify an input file or an input directory
...
```

**Comment by bgruening** [comment]
@RalfG if you like you can add here a symlink: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/thermorawfileparser/meta.yaml#L15

**Comment by RalfG** [comment]
Symlink for the lowercase sounds good. I guess no one would be against a secondary name. But what about the .sh extension? Any opinions?

**Comment by bgruening** [comment]
> thermorawfileparser

I think this is fine.

**Comment by caetera** [comment]
Just wonder, can this issue be closed? Was it solved?

**Comment by caetera** [comment]
Since you know about all this conda stuff, do you have any suggestions for #85 ?

**Comment by RalfG** [comment]
@caetera, yes, this has been merged in bioconda/bioconda-recipes#23142, so this can be closed. 

I definitely don't know all 😛 But I'll take a look.
